stories/input.stories.js

```diff
@@ -0,0 +1,236 @@
+import React from 'react';
+import { Story, Demo, Description, Code, comment } from './storybook/blocks';
+import './input.scss';
+
+export default { title: 'Input' };
+
+export const input = () => (
+  <Story>
+    <Demo>
+      <div>
+        <div className="Manifold__Input">
+          <label for="my-input">Text input</label>
+          <input
+            type="text"
+            autoCapitalize="off"
+            name="text-input"
+            id="my-input"
+            placeholder="Placeholder"
+          />
+        </div>
+      </div>
+      <div>
+        <div className="Manifold__Input">
+          <label for="email-input">Email</label>
+          <input type="email" name="email-input" id="email-input" value="info@manifold.co" />
+        </div>
+      </div>
+      <div>
+        <div className="Manifold__Input">
+          <label for="password-input">Password</label>
+          <input type="password" name="password-input" id="password-input" />
+        </div>
+      </div>
+      <div>
+        <div className="Manifold__Input">
+          <label for="number-input">Numeric</label>
+          <input
+            type="text"
+            inputmode="numeric"
+            pattern="[0-9]*"
+            name="number-input"
+            id="number-input"
+          />
+        </div>
+      </div>
+    </Demo>
+    <Description>
+      <h1>Inputs</h1>
+      <p>
+        Standard <code>&lt;input&gt;</code> tags, from text to email to password to numeric inputs.
+      </p>
+    </Description>
+    <Code
+      tabs={{
+        html: `<!-- text input -->
```

Comment:
I could see this becoming difficult to maintain. It would be cool to automate this. If we kept this HTML in a neighboring file we could import it as raw text. Then we could add it here and also insert the HTML into the story with `dangerouslySetInnerHTML`.

Reply:
+1. Yeah agreed. For now, this is an experiment, because there are points where we want to maybe have a little more going on in the “Demo” above than what we show in the HTML down here. So I’m still figuring that out.

But agreed the editing experience could be better. I’ll think about what patterns emerge as we have more of these components to deal with.